### ISSUE TITLE
why are you backpropagating the mean of the mean of the loss?

### ISSUE THREAD

**Issue by fawazsammani** [ruotianluo/self-critical.pytorch#150]
Sorry. I figured out that my question is stupid. Didn't pay attention well.
